blockly/frontend/src/variableList.ts

```diff
@@ -0,0 +1,149 @@
+const DEFAULT_VALUE = 'NO SET';
```

Comment:
```suggestion
const DEFAULT_VALUE = 'Nicht gesetzt';
```

Reply:
```suggestion
const DEFAULT_VALUE = '?';
```

Ist das evtl. schöner?

Reply:
i like it